server/routes/courtRegister/courtRegisterController.test.ts

```diff
@@ -163,7 +161,7 @@ describe('Court Register controller', () => {
   })
   describe('Add new court flow', () => {
     beforeEach(() => {
-      courtRegisterService = new CourtRegisterService(null) as jest.Mocked<CourtRegisterService>
+      courtRegisterService = new CourtRegisterService({} as HmppsAuthClient) as jest.Mocked<CourtRegisterService>
```

Comment:
was wondering whether might make sense to extract out a mock services for this - I find this having to call the constructor really annoying.

Reply:
@andrewrlee what did you mean `extract out` ? not use Jest or move the `new CourtRegisterService({} as HmppsAuthClient) ` to a different file?